.github/workflows/e2e.yml

```diff
@@ -0,0 +1,104 @@
+name: E2E Test
```

Comment:
Jules name this test `Interactive Reverse Shell REPL Test` and update the filename accordingly